doc/nrf/drivers/entropy_cc310.rst

```diff
@@ -7,18 +7,19 @@ CC310 entropy driver
    :local:
    :depth: 2
 
-You can use the CC3xx entropy driver (entropy_cc310) to generate random data using the Arm CryptoCell CC310 hardware.
-This hardware is available on the nRF52840 and nRF9160 devices.
+You can use the CC3xx entropy driver (*entropy_cc310*) to generate random data using the Arm CryptoCell CC310 hardware available on the following Nordic SoCs:
 
-When used on nRF52840, the entropy_cc3xx driver gathers entropy by using the CC310 hardware through the :ref:`nrf_cc3xx_platform_readme`.
+* nRF52840
+* nRF5340
+* nRF9160
```

Comment:
I'm tempted to suggest using sample_board_rows here, but it might be a bit too much information...